src/components/modules/epics/ContributionsEpicReminder.tsx

```diff
@@ -120,9 +119,7 @@ type ReminderPayload = {
     reminderDate: string;
 };
 
-const contributionsReminderUrl = isProd
```

Comment:
this is being called from inside a component, so we shouldn't rely on environment variables